dojo/tools/aws_prowler_v3/parser.py

```diff
@@ -0,0 +1,104 @@
+
+import hashlib
+import json
+import textwrap
+from datetime import date
+
+from dojo.models import Finding
+
+
+class AWSProwlerJsonV3Parser(object):
+    SCAN_TYPE = ["AWS Prowler V3"]
+
+    def get_scan_types(self):
+        return AWSProwlerJsonV3Parser.SCAN_TYPE
+
+    def get_label_for_scan_types(self, scan_type):
+        return AWSProwlerJsonV3Parser.SCAN_TYPE[0]
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Export of AWS Prowler JSON V3 format."
+
+    def get_findings(self, file, test):
+        if file.name.lower().endswith('.json'):
+            return self.process_json(file, test)
+        else:
+            raise ValueError('Unknown file format')
+
+    def process_json(self, file, test):
+        dupes = dict()
+
+        data = json.load(file)
+        for deserialized in data:
+
+            status = deserialized.get('Status')
+            if status.upper() != 'FAIL':
+                continue
+
+            account = deserialized.get('AccountId')
+            region = deserialized.get('Region')
+            provider = deserialized.get('Provider')
+            compliance = str(deserialized.get('Compliance'))
+            result_extended = deserialized.get('StatusExtended')
+            general_description = deserialized.get('Description')
+            asff_compliance_type = " / ".join(deserialized.get('CheckType'))
+            severity = deserialized.get('Severity').capitalize()
+            aws_service_name = deserialized.get('ServiceName')
+            impact = deserialized.get('Risk')
+            mitigation = deserialized.get('Remediation').get('Recommendation').get("Text")
+            mitigation = str(mitigation) + "\n" + str(deserialized.get('Remediation').get('Code'))
```

Comment:
Using chained `.get()` can end poorly if the expected field is not found. For example if `Remediation` is not present, the line becomes `str(None.get("Code"))` and would throw an exception. Please update all other references of `.get()` chains
```suggestion
            mitigation = str(mitigation) + "\n" + str(deserialized.get('Remediation', {}).get('Code', ''))
```